groups/tests/test_managers.py

```diff
@@ -0,0 +1,108 @@
+from django.test import TestCase
+from incuna_test_utils.compat import Python2AssertMixin
+
+from . import factories
+from .. import models
+
+
+class TestGroupManager(Python2AssertMixin, TestCase):
+    def test_discussions(self):
+        discussion = factories.DiscussionFactory.create()
+        discussion_two = factories.DiscussionFactory.create()  # Has a different group
```

Comment:
Does it matter that the `group` is different? If so it shouldn't be implicit in the `DiscussionFactory` definition.